frontend/src/components/common/podcasts-list/podcasts-list.tsx

```diff
@@ -1,17 +1,23 @@
 import { Podcast as TPodcast } from 'common/types/types';
-import { PodcastItem } from 'components/common/common';
+import { PodcastItem, Button } from 'components/common/common';
+import { ButtonColor } from 'common/enums/enums';
 import styles from './styles.module.scss';
 
 type Props = {
   podcasts: TPodcast[];
+  isLoading?: boolean;
+  onMorePodcastsLoad?: () => void;
```

Comment:
podcast list - just a component for render
this component shouldn't know anything about pagination